boards/hifive1/include/periph_conf.h

```diff
@@ -32,6 +32,7 @@ extern "C" {
  *
  * @{
  */
+#define CLOCK_TIMERCLOCK            (RTC_FREQ)
```

Comment:
How is this configurable?

Reply:
It's not, the hifive boards use the RTC oscillator here (32.768 Khz), the qemu emulation target has it configured to 10 MHz.

There is an [upstream patch](https://lists.nongnu.org/archive/html/qemu-riscv/2020-11/msg00013.html) to set it to 32.768 KHz, matching the real hardware.

Reply:
I asked because the commit message says `fe310: Make timer frequency configurable ` which I did not quite undersand.

Reply:
Ahhh, I can rephrase that commit. The intention of the commit is to not have the `periph/timer.c` hardcode the timer value to `RTC_FREQ`